### ISSUE TITLE
Nothing seems to be happening

### ISSUE THREAD

**Issue by RussMorg** [kylereicks/auto-summary#1]
Installed the plugin, created a post, but no summary exists in the excerpt.

Where have I gone wrong?  Is it necessary to add the mentioned filters, or should it just work with the default settings?

Many thanks for the plugin though, it'll be great if I can get it to work.

**Comment by kylereicks** [comment]
Thanks @RussMorg . I see that the script enqueues required the plugin directory to be named "auto-summary", which may not be the case depending on how the plugin was installed. I pushed an [update](https://github.com/kylereicks/auto-summary/commit/4748a4f4bba0927bd6807f50efebab2c9a19200a)  that registers the resources relative to the plugin's directory for more flexibility.
If you are able to try re-installing with the latest updates from the master branch, I think things might work better.

**Comment by RussMorg** [comment]
Okay @kylereicks I removed the old and installed the new, and while 'Use auto-summary' now appears in the extract meta box, the button doesn't seem to do anything.  I tried pasting the article into the top box and pressing 'Use auto-summary', and it just empties the top box, nothing gets generated.

How should this plugin work?  Should the summary be created automatically when the post is published, or is the idea that you write the post, and then manually hit the 'Use auto-summary' button to create a summarised excerpt?
